components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+      fn convert_i64_to_string(path: &str, json: &str) -> String;
+      fn convert_string_to_i64(path: &str, json: &str) -> String;
+    }
+}
+
+// Converts int64/uint64 value to string and serializes json with new values.
+// A path is a Unicode string with the reference tokens separated by /. Inside tokens / is replaced by ~1 and ~ is replaced by ~0.
+// Examples: convert_i64_to_string("/a/b", json) for { a : { b : 1 }}
+//           convert_i64_to_string("/a/0", json) for { a : [ 1 ]}
+pub fn convert_i64_to_string(path: &str, json: &str) -> String {
+  let mut unwrapped_value: serde_json::Value = serde_json::from_str(&json).unwrap_or_else(|e| {
+    eprintln!("Failed to parse JSON: {}", e);
+    return serde_json::Value::Null;
+  });
+  if unwrapped_value.is_null() {
+    return String::new();
+  }
+  let mutable_pointer = unwrapped_value.pointer_mut(path);
+  if mutable_pointer.is_none() {
+    return String::new();
+  }
+  let value = mutable_pointer.unwrap();
+  if value.is_null() {
+    return String::new();
+  }
+  let valid_type = value.is_u64() | value.is_i64();
```

Comment:
re uint vs int above, it doesn't seem safe to try to handle both cases in one method because you can't differentiate between them based on value alone except at the edges. I think it should be explicit `convert_uint64_to_unicode_string` and only do int64 if we actually need it for something right now

Reply:
done